secantTests/BalanceBreakdownTests/BalanceBreakdownTests.swift

```diff
@@ -28,7 +28,9 @@ class BalanceBreakdownTests: XCTestCase {
         store.dependencies.mainQueue = .immediate
         store.dependencies.numberFormatter = .noOp
         
-        await store.send(.onAppear)
+        await store.send(.onAppear) { state in
+            state.autoShieldingThreshold = Zatoshi(100_000)
```

Comment:
We're using the shielding threshold across several files. Could we move it into a constant?

Reply:
Updated